includes/Plugin.php

```diff
@@ -172,6 +180,7 @@ public function register() {
 		// Plugin compatibility / polyfills.
 		add_action( 'wp', [ $this, 'load_amp_plugin_compat' ] );
 		add_action( 'init', [ $this, 'includes' ] );
+		add_action( 'rest_dispatch_request', [ $this, 'load_amp_plugin_compat' ] );
```

Comment:
Now that this is merged - https://github.com/google/web-stories-wp/pull/5868, please remove this workaround and rebase.